lib/python/pathling/__init__.py

```diff
@@ -49,4 +51,6 @@
     "to_snomed_coding",
     "to_ecl_value_set",
     "find_jar",
+    "exp",
+    "Expression",
```

Comment:
We should re-export all of the new imports, so that the user can import all the things from Pathling in one line.

Reply:
yes